.github/workflows/checks.yml

```
@@ -31,8 +31,14 @@ jobs:
       - name: 🔨 Build
         run: cargo build --verbose
 
+      - name: Install latest nextest release
+        uses: taiki-e/install-action@nextest
+
       - name: 🧪 Run tests
-        run: cargo test --verbose
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.4)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)